packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -102,15 +103,15 @@
   }
 
   .nav-burger {
-    display: block;
```

Comment:
this change fix the burger button issue. It wasn't displaying at 992px screen. Figured out we were referring two different ways min-width and max-width to switch styles.